.github/ISSUE_TEMPLATE/change.yml

```diff
@@ -0,0 +1,10 @@
+name: Change Request
```

Comment:
You might also add a support section for like validations or something 🤷

Reply:
That's a good idea, but off the top of my head I'm not sure exactly what info we'll want for new validation requests. I'm inclined to leave that as a generic change request for now and creating a specific form once we get requests and figure out what info makes since.